encrypted-media-respec.html

```diff
@@ -2332,8 +2332,8 @@
         <section>
           <h4><a>MediaKeyMessageEventInit</a></h4>
           <div><pre class="idl">[SecureContext] dictionary MediaKeyMessageEventInit : EventInit {
-             MediaKeyMessageType messageType = "license-request";
-             ArrayBuffer         message;
+             required MediaKeyMessageType messageType;
```

Comment:
This is OK if you don't think there's any reasonable default. There's not a single hit for `MediaKeyMessageEventInit` in HTTP Archive, so I think legacy compat is not a problem.